arq/connections.py

```diff
@@ -43,8 +44,24 @@ def __repr__(self):
 class ArqRedis(Redis):
     """
     Thin subclass of ``aioredis.Redis`` which adds :func:`arq.connections.enqueue_job`.
+
+    :param redis_settings: an instance of ``arq.connections.RedisSettings``.
+    :param _serialize: a function that serializes Python objects to bytes, defaults to pickle.dumps
+    :param _deserialize: a function that deserializes bytes into Python objects, defaults to pickle.loads
+    :param kwargs: keyword arguments directly passed to ``aioredis.Redis``.
     """
 
+    def __init__(
+        self,
+        pool_or_conn,
+        _serialize: Optional[Callable[[Any], bytes]] = None,
```

Comment:
```suggestion
        _job_serializer: Optional[Callable[[Any], bytes]] = None,
```
Might be even clearer?

Reply:
and same obviously for `_deserialize` -> `_job_deserializer`.